tests/test-module.py

```diff
@@ -48,12 +50,18 @@ def test_returns_remote_s3_object(self):
         """ Test remote s3 event is returned when 'replace' key is present """
         result = self.cumulus_message_adapter.loadRemoteEvent(self.event_with_replace)
         assert result == self.s3_object
-
+        
```

Comment:
Guessing this added whitespace was accidental?

Reply:
Indeed.  I'm apparently relying on auto whitespace removal a bit too much.

Reply:
26e58af